linker/shared/src/test/scala/org/scalajs/linker/testutils/TestIRBuilder.scala

```diff
@@ -134,4 +134,5 @@ object TestIRBuilder {
     MethodIdent(name)
 
   def int(x: Int): IntLiteral = IntLiteral(x)
+  def str(x: String): StringLiteral = StringLiteral(x)
```

Comment:
Same comment then in the other PR to use this more widely :)

Reply:
Since it's done in the other PR, I'll leave this one as is. I'll rebase this one once #4418 is merged, so this line will disappear from this commit.